Source/Charts/Utils/Platform.swift

```diff
@@ -139,6 +139,14 @@ public typealias NSUIGestureRecognizerState = UIGestureRecognizer.State
 		}
 	}
 
+	extension UIColor
+	{
+		static var labelColor : UIColor
+		{
+			return UIColor.black
```

Comment:
I get that this is an old PR, but if UIColor.label is returned here, we can get Dark Mode support for labels on iOS as well, which would be nice.

Reply:
Simply removing the declaration will solve the issue for iOS 13+.
```
extension UIColor {
    @available(iOS, introduced: 9.0, obsoleted: 13.0)
    @available(tvOS, introduced: 9.0, obsoleted: 13.0)
    static var label: UIColor {
        return .black
    }
}
```